proseco/acq.py

```diff
@@ -603,7 +603,7 @@ def get_initial_catalog(self):
                     # Find the position in box_sizes that corresponds to include_id
                     # and set to the specified include_halfw.
                     for idx in range(len(acq_indices)):
-                        if include_id == cand_acqs[idx]['id']:
+                        if include_id == cand_acqs[acq_indices[idx]]['id']:
```

Comment:
This looks fine.  I got a bit lost in why this section had a double loop to try to find matches but I'm good with the more minimal change.

Reply:
The double loop was to try keeping the logic simple.  This morning I tried and failed to understand your implementation, brain wasn't firing enough.  But when I looked at the original code the bug and solution was immediately obvious to me.  Not surprising since I wrote the original...

Reply:
Indeed I think the reason I couldn't quite wrap my head around your implementation was the same "brain firing enough" issue.  I'm still trying to figure out when in the pandemic day I have the most brain power!  Thanks!